CMakeLists.txt

```diff
@@ -46,6 +46,15 @@ include(FindThreads)
 option(BUILD_LIBPMEMFILE
 	"build libpmemfile (requires libsyscall_intercept)" ON)
 
+option(BUILD_LIBPMEMFILE_TESTS
+	"build libpmemfile tests (ignored if BUILD_LIBPMEMFILE is not set)" ON)
+
+if(BUILD_LIBPMEMFILE AND BUILD_LIBPMEMFILE_TESTS)
+	set(ACTUALLY_BUILD_LIBPMEMFILE_TESTS ON)
```

Comment:
With
```
if (NOT BUILD_LIBPMEMFILE)
   set(BUILD_LIBPMEMFILE_TESTS OFF)
endif()
```
you won't need ACTUALLY_BUILD_LIBPMEMFILE_TESTS.

Reply:
You can't just override an option set by the user (I tried it, it just stays ON).
But I'll try to use CMakeDependentOption instead.

Reply:
Done